runtime/src/core/closures.js

```diff
@@ -17,19 +15,10 @@ export class GrainClosure {
   }
 
   jsFunc(...args) {
-    if (args.length != this.arity) {
-      throwGrainError(
-        GRAIN_ERR_ARITY_MISMATCH,
-        this.arity,
-        args.length,
-        this.runtime
-      );
-    } else {
-      let grainVals = args.map((x) => JSToGrainVal(x, this.runtime));
-      grainVals.unshift(this.loc * 4);
-      // [TODO]: We should feed in a runtime here.
-      return grainToJSVal(null, this.func(...grainVals));
-    }
+    let grainVals = args.map((x) => JSToGrainVal(x, this.runtime));
+    grainVals.unshift(this.loc * 4);
+    // [TODO]: We should feed in a runtime here.
```

Comment:
Issue for this?

Reply:
Only annoyed because it's not my TODO 😛 
https://github.com/grain-lang/grain/issues/578